### PR TITLE
fix: receivers parameters

### PR DESCRIPTION
**Pull request by jma**
Signed-off-by: Johnny Mariéthoz <Johnny.Mariethoz@rero.ch>

### CODE REVIEW

File: invenio_collections/receivers.py [hunk 1]
```diff
@@ -106,7 +106,7 @@ def __init__(self, app=None):
             from .percolator import _find_matching_collections_externally
             self.matcher = _find_matching_collections_externally
 
-    def __call__(self, record, **kwargs):
+    def __call__(self, sender, record, **kwargs):
```
Comment (jacquerie) [inline invenio_collections/receivers.py]: As Travis says, you will also need to amend the tests correspondingly. But I want to know why is this change needed: can you point me to the signature of the signal?

Comment (lnielsen) [reply invenio_collections/receivers.py]: It's probably due to https://github.com/inveniosoftware/invenio-records/commit/cd3a7d2660e20f00063d142db32fba1ffec2fe6b (backward incompatible change)

Comment (lnielsen) [inline invenio_collections/receivers.py]: it should probably be `` ..., sender, record=None, ...``

Comment (lnielsen) [reply invenio_collections/receivers.py]: And you might need to bump lowest invenio-records dependency
